pkg/cli/cmds/agent.go

```diff
@@ -54,8 +54,16 @@ func NewAgentCommand() cli.Command {
 }
 
 func AgentRun(clx *cli.Context) error {
-	if profile == "" {
+	switch profile {
+	case "cis-1.5":
+		if err := validateCISreqs(); err != nil {
```

Comment:
why did you need to move the validateCISreqs() logic?

Part of the reason I ask is that I think you are oging to have to change it to fix the etcd user bug i mentioned

Reply:
answered offline and the TL;DR is that this needed moved once the arg was moved out to the server and agent commands